src/styles/news-post.css

```diff
@@ -0,0 +1,262 @@
+.excerpt {
+  @apply text-palette-300 my-4 text-lg font-light;
+}
+
+a {
+  @apply text-primary hover:text-primary-light transition-colors;
+}
+
+.content-body {
```

Comment:
can you add a comment here that says that this is styling that applies to articles sourced from ghost? content-body is pretty generic sounding class

Reply:
[525e8b1](https://github.com/SkynetLabs/skynetlabs-website/pull/14/commits/525e8b1a8ad5505f4540f65d949f2b9ff81c1433)